### DISCUSSION TITLE
[Suggestion] Add the storage_block tag to the food item crates

### DISCUSSION THREAD

**Discussion by ffuentesm** [vectorwing/FarmersDelight#750] [Ideas]
The idea is to add the storage_block tag to the food crates as they are mechanically storage blocks.
This would help unify some duplicated recipes, such as the food crates from Thermal and Quark.

**Comment by Brokensweet** [comment]
How can you download this I have tried so many times

**Comment by ffuentesm** [comment]
> How can you download this I have tried so many times

What do you mean? Can you elaborate?

**Comment by vectorwing** [comment]
`storage_blocks` is a Forge tag, right? I see it also has sub-folders to it.

I'll take a look later at how Quark and Thermal Cultivation are handling it, just to make sure I dont mess it up. Sounds like a reasonable tag to use.

**Comment by ffuentesm** [comment]
The crate blocks have the forge:storage_block{material} tag? So they are in line with other storage blocks of the mod, such as the sugarcane, bamboo block, block of charcoal. You just need to add the item vanilla id to the subfolder. 
Thermal already has sugarcane and bamboo, with charcoal to add to it. 
Supplementaries add sugar and flint storage block format with that tag.

**Comment by ffuentesm** [comment]
I forgot, but was this ever implemented? 
Now that 1.21 came out, I noticed that the Storage Crates have no tags still. 
I think Quark and Thermal did add the Storage Block tag in their 1.20.1 versions and will carry over to 1.21 when they release it. 

Just like how the Hay Bale has the `c:storage_blocks` tag, it would make sense if something like the Straw Bale had it too. (and the other crates and bags too)
![image](https://github.com/user-attachments/assets/90c119c2-5cc4-4207-9602-953ac031c9b7)
